File: PHP_TP2/EntitasBioskop.php
<?php
class EntitasBioskop {
    protected string $id;
    protected string $nama;
    protected int $tahunOperasional;

    public function __construct(string $id, string $nama, int $tahunOperasional) {
        $this->id = $id;
        $this->nama = $nama;
        $this->tahunOperasional = $tahunOperasional;
    }

    public function getId(): string { return $this->id; }
    public function setId(string $id): void { $this->id = $id; }

    public function getNama(): string { return $this->nama; }
    public function setNama(string $nama): void { $this->nama = $nama; }

    public function getTahunOperasional(): int { return $this->tahunOperasional; }
    public function setTahunOperasional(int $tahun): void { $this->tahunOperasional = $tahun; }
}
?>

File: CPP/EntitasBioskop.cpp
#include <iostream>
#include <string>

using namespace std;

class EntitasBioskop {
protected:
    string id;
    string nama;
    int tahunOperasional;

public:
    EntitasBioskop() {}
    EntitasBioskop(string id, string nama, int tahunOperasional) {
        this->id = id;
        this->nama = nama;
        this->tahunOperasional = tahunOperasional;
    }

    // Getter & Setter
    string getId() const { return id; }
    void setId(string id) { this->id = id; }

    string getNama() const { return nama; }
    void setNama(string nama) { this->nama = nama; }

    int getTahunOperasional() const { return tahunOperasional; }
    void setTahunOperasional(int tahunOperasional) { this->tahunOperasional = tahunOperasional; }

    virtual ~EntitasBioskop() {}
};